src/Engine/System/TimedSystem.hpp

```diff
@@ -31,25 +25,136 @@ class RA_ENGINE_API AbstractTimedSystem : public System
     virtual bool restoreFrame( const std::string& dir, uint frameID ) = 0;
 };
 
-class RA_ENGINE_API CoupledTimedSystem : public BaseCouplingSystem<AbstractTimedSystem>
+/**
+ * The TimeSystem is the System managing time.
+ * It dispatches time-related events to its AbstractTimedSystem.
+ */
+class RA_ENGINE_API TimeSystem : public BaseCouplingSystem<AbstractTimedSystem>
 {
   public:
-    void play( bool on ) override {
-        dispatch( [on]( const auto& s ) { s->play( on ); } );
-    }
-    void step() override {
-        dispatch( []( const auto& s ) { s->step(); } );
-    }
-    void reset() override {
-        dispatch( []( const auto& s ) { s->reset(); } );
-    }
-    void cacheFrame( const std::string& dir, uint frameID ) const override {
-        dispatch( [&dir, frameID]( const auto& s ) { s->cacheFrame( dir, frameID ); } );
-    }
-    bool restoreFrame( const std::string& dir, uint frameID ) override {
-        return conditionnaldispatch(
-            [&dir, frameID]( const auto& s ) { return s->restoreFrame( dir, frameID ); } );
-    }
+    /**
+     * Builds a TimeSystem with the given time delta \p dt and maximal frame
+     * \p playRange.
```

Comment:
playRange is not a parameter of TimeSystem constructor.
use \param doxygen command to document parameter rôle. and whatch out doxygen warnings and error when generating documentation.

Reply:
updated the doc